specification/compute/resource-manager/Microsoft.Compute/stable/2018-10-01/compute.json

```diff
@@ -6071,6 +6074,23 @@
       },
       "description": "Specifies the operating system settings for the virtual machine."
     },
+    "AutomaticRepairsPolicy": {
+      "properties": {
+        "enabled": {
```

Comment:
Is this optional property?  If so, if enabled is not given, it is interpreted as enabled=false in service?